lua/obsidian/command.lua

```diff
@@ -647,6 +647,33 @@ command.check_health = function(client, _)
   end
 end
 
+command.switch_workspace = function(client, data)
+  if not data.args or #data.args == 0 then
+    echo.info(
+      "Current workspace: " .. client.current_workspace.name .. " @ " .. tostring(client.dir),
+      client.opts.log_level
+    )
+    return
+  end
+
+  local workspace = nil
+  for _, value in pairs(client.opts.workspaces) do
+    if data.args == value.name then
+      workspace = value
+    end
+  end
+
+  if not workspace then
+    echo.err("Workspace '" .. data.args .. "' does not exist", client.opts.log_level)
+    return
+  end
+
+  client.current_workspace = workspace
+
+  echo.info("Switching to workspace '" .. workspace.name .. "' (" .. workspace.path .. ")", client.opts.log_level)
+  client.dir = Path:new(workspace.path)
```

Comment:
Same suggestion here about normalizing:

```suggestion
  client.dir = Path:new(vim.fs.normalize(tostring(workspace.path)))
```

Reply:
See [my comment](https://github.com/epwalsh/obsidian.nvim/pull/155#discussion_r1361333890) in a previous suggestion

Reply:
Fixed with 598bc0a